### PR TITLE
Add a configuration option for inverted mouse wheel zoom

### PR DESCRIPTION
**Pull request by JackCrumpLeys**
#479

### CODE REVIEW

File: crates/camera/src/camera.rs [hunk 1]
```diff
@@ -274,7 +274,12 @@ impl DesiredDistance {
     }
 
     fn zoom_clamped(&mut self, conf: &CameraConf, factor: f32) {
-        self.0 = (self.0 * factor).clamp(conf.min_distance(), conf.max_distance());
+        self.0 = (if conf.scroll_inverted() {
```
Comment (JackCrumpLeys) [inline crates/camera/src/camera.rs:277]: Is this the best place for the logic?

Comment (Indy2222) [reply crates/camera/src/camera.rs:277]: I think so.

**Review by Indy2222 — CHANGES_REQUESTED** [review]
Please update the documentation as part of the implementation commits. Otherwise LGTM.

**Review by Indy2222 — COMMENTED** [review]
Please squash the two commits and add `Closes #479.` to commit description. Otherwise LGTM.

**Review by Indy2222 — APPROVED** [review]

**Pull request merged**